smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/DirectedTypeScriptCodegen.java

```diff
@@ -102,6 +102,11 @@ public TypeScriptCodegenContext createContext(CreateContextDirective<TypeScriptS
             });
         });
 
+        directive.integrations().forEach(integration -> {
+            LOGGER.info(() -> "Mutating plugins from TypeScriptIntegration: " + integration.getClass().getName());
```

Comment:
nit: Integrations have a `name()` method. I would recommend using that here over the class name.